ts-client/src/dlmm/index.ts

```diff
@@ -4284,6 +4286,168 @@ export class DLMM {
     };
   }
 
+  /**
+ * The `seedLiquidity` function create multiple grouped instructions. The grouped instructions will be either [initialize bin array + initialize position instructions] or [deposit instruction] combination.
+ * @param
+ *    - `owner`: The public key of the positions owner.
+ *    - `seedAmount`: Lamport amount to be seeded to the pool.
+ *    - `price`: Price in UI format
+ *    - `base`: Base key
+ *    - `roundingUp`: 
+ * @returns {Promise<Transaction>}
+ */
+  public async seedLiquiditySingleBin(
+    owner: PublicKey,
+    base: PublicKey,
+    seedAmount: BN,
+    price: number,
+    roundingUp: boolean
+  ): Promise<Transaction> {
+    const toLamportMultiplier = new Decimal(
+      10 ** (this.tokenY.decimal - this.tokenX.decimal)
+    );
+
+    const pricePerLamport = new Decimal(price).mul(toLamportMultiplier);
+    let binId: BN = DLMM.getBinIdFromPrice(pricePerLamport, this.lbPair.binStep, true);
+    if (roundingUp) {
+      binId = DLMM.getBinIdFromPrice(pricePerLamport, this.lbPair.roundStep, false);
+    }
+
+    if (binId != this.lbPair.activeId) {
+      throw new Error(`binId doesn't match active bin ID`);
+    }
+
+    const lowerBinArrayIndex = binIdToBinArrayIndex(binId);
+    const upperBinArrayIndex = lowerBinArrayIndex.add(new BN(1));
+
+    const [lowerBinArray] = deriveBinArray(this.pubkey, lowerBinArrayIndex, this.program.programId);
+    const [upperBinArray] = deriveBinArray(this.pubkey, upperBinArrayIndex, this.program.programId);
+    const [positionPda] = derivePosition(this.pubkey, base, binId, 1, this.program.programId);
+
+    const setComputeUnitLimitIx = computeBudgetIx();
```

Comment:
I dont think that tx will consume max 1.4M

Reply:
okay will remove the set CU limit ix